src/tools/importer/README.md

```diff
@@ -1,33 +1,149 @@
-column         | c            | vertex and edge's column.
-file           | f            | data file path.
-help           | h            | help message.
-ranking        | k            | the edge have ranking data.
-schema         | m            | specify the schema name.
-name           | n            | specify the space name.
-timeout        | o            | thrift connection timeout.
-pswd           | p            | thrift service password.
-retry          | r            | thrift connection retry number.
-stat           | s            | print statistics info.
-type           | t            | data type. vertex or edge.
-user           | u            | thrift service username.
+#### Install JARs
+
+```
+ > mvn install:install-file -Dfile=$YOUR_GRAPH_CLIENT_FILEPATH -DgroupId=nebula-graph -DartifactId=graph-client -Dversion=$GRAPH_CLIENT_VERSION -Dpackaging=jar
+
+ > mvn install:install-file -Dfile=$YOUR_FBTHRIFT_FILEPATH -DgroupId=com.facebook -DartifactId=thrift -Dversion=$FBTHRIFT_VERSION -Dpackaging=jar
+```
+
+Replace "$YOUR_GRAPH_CLIENT_FILEPATH" and "$YOUR_FBTHRIFT_FILEPATH" with your own values. Also "$GRAPH_CLIENT_VERSION" and "$FBTHRIFT_VERSION".
+
+### Get Importer
+
+At this moment, please contact us to get the `importer` jar package.
+
+Or build it by yourself.
+
+```
+> cd $YOUR_NEBULA_DIR
+> cd src/tools/importer
+> mvn clean package
+```
+
+The `importer` jar package is under the target dirctory.
+
+### Data File
+
+#### For Tags
+
+The Impoter can only import one tag or edgetype per time. For tags, the first column is `vid` and then the property value list, separated by a comma. 
+
+```
+vid,prop_name_1,prop_name_2,...
+```
+
+#### For Edge Types
+
+For edge type, the file structure is as following:
+
+```
+sourceID, destinationID, [ranking], prop_name_1, prop_name_2, ...
+```
+
+> The "ranking" column is optional. It works with "--ranking" optional, which will be described below.
+
+### Import Data
+
+To import data, run:
+
+```
+> java -jar importer-1.0.0-beta.jar <OPTIONS>
+```
+
+|Property Name  | Abbreviation |  Description| Example |
+|:----|:----:|:----:|:----:|
+|--address        | -a            | graphd service address.| 127.0.0.1:3699 |
+|--batch          | -b            | batch insert size.|2|
+|--name           | -n            | specify the space name.| myspace_test2 |
+|--schema         | -m            | specify the tag or edgetype name.| student |
+|--column         | -c            | properties of tag or edge to be inserted, separated by a comma | name,age |
+|--ranking        | -k            | the edge have ranking data. Default is false| true/false|
+|--file           | -f            | data file| ./tmp/data.txt |
+|--errorPath    | -d            | error log file | ./tmp/error.log |
+|--help           | -h            | list help||
+|--timeout        | -o            | specify connection timeout, in millisecond| 3000 |
+|--pswd           | -p            | graphd service password||
```

Comment:
I think there is an unwanted `|` here although it won't do any harm to the table's looking.